repo_checker.py

```diff
@@ -278,52 +277,27 @@ def install_check_parse(self, output):
         if section:
             yield InstallSection(section, text)
 
-    @memoize(ttl=60, session=True)
-    def cycle_check_skip(self, project):
```

Comment:
When new cycles are introduced intentionally how will someone indicate that to repo-checker?

Reply:
by changing the config.